### PR TITLE
Update Harbormaster status when using direct Taskcluster tasks

### PR DESCRIPTION
**Pull request by La0**
- ~~Send Taskcluster link to Phabricator~~ :crying_cat_face: :it: 
- Provides the Harbormaster Target PHID to the SA task to avoid looking for it (and potentially missing it)

### CODE REVIEW

**Comment by marco-c** [comment]
We could just upload the build status, without the link. This way we avoid changing the UI twice (now and in a bit when we'll use try).

File: src/pulselistener/pulselistener/listener.py [hunk 1]
```diff
@@ -26,6 +26,8 @@
 MODE_PHABRICATOR_POLLING = 'polling'
 MODE_PHABRICATOR_WEBHOOK = 'webhook'
 
+TASK_URL = 'https://tools.taskcluster.net/task-inspector/#{}'
```
Comment (marco-c) [inline src/pulselistener/pulselistener/listener.py]: This is unused now :P

File: src/staticanalysis/bot/static_analysis_bot/revisions.py [hunk 2]
```diff
@@ -233,13 +233,15 @@ def load_phabricator(self, diff_phid):
         self.id = self.revision['id']
 
         # Load build for status updates
-        if settings.build_plan:
+        if 'HARBORMASTER_TARGET' in os.environ:
+            self.build_target_phid = os.environ['HARBORMASTER_TARGET']
+        elif settings.build_plan:
```
Comment (marco-c) [inline src/staticanalysis/bot/static_analysis_bot/revisions.py]: Maybe we can remove this branch altogether now?

Comment (La0) [reply src/staticanalysis/bot/static_analysis_bot/revisions.py]: I don't think so yet: in remote+Try mode, there is no env variable for specifying the Target PHID.
It's another PR to do, but mostly in mozilla central

**Pull request merged**